client/cmd/install.go

```diff
@@ -370,18 +382,18 @@ func isZshConfigured(homedir string) (bool, error) {
 	if errors.Is(err, os.ErrNotExist) {
 		return false, nil
 	}
-	bashrc, err := os.ReadFile(getZshRcPath(homedir))
+	zshrc, err := os.ReadFile(getZshRcPath(homedir))
 	if err != nil {
 		return false, fmt.Errorf("failed to read zshrc: %w", err)
 	}
-	return strings.Contains(string(bashrc), getZshConfigFragment(homedir)), nil
+	return strings.Contains(string(zshrc), getZshConfigFragment(homedir)), nil
 }
 
 func getBashConfigPath(homedir string) string {
 	return path.Join(homedir, data.GetHishtoryPath(), "config.sh")
 }
 
-func configureBashrc(homedir, binaryPath string, skipConfigModification bool) error {
+func configureBashrc(homedir, binaryPath string, skipConfigModification bool) (err error) {
```

Comment:
Could you share a bit about the reasoning behind this change? I'm curious because I don't see a clear difference from the previous version in that case.

It seems like it would be better to keep it consistent with the rest of the codebase, it improves readability. Small inconsistencies can sometimes make other developers pause and wonder why. What do you think?

Reply:
I agree that it could be changed universally, if agreed on as consistent style. I wouldn't mind doing that all in 1 commit changing all functions to that style

Its simply much easier to code golang funcs like that with named returns, cuz then you don't have to instantiate the err variable and you don't need to keep track of where it is instantiated.